repos/fdbt-site/src/components/TapPricingRow.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactElement } from 'react';
+import { ErrorInfo, MultiTapPricing } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from './FormElementWrapper';
+
+interface TapPricingRowProps {
+    numberOfTapsToDisplay: number;
+    errors: ErrorInfo[];
+    userInput: MultiTapPricing[];
+}
+
+export const renderTable = (index: number, errors: ErrorInfo[], userInput: MultiTapPricing[] = []): ReactElement => (
+    <fieldset key={index} className="govuk-fieldset">
+        <legend className="govuk-fieldset__legend govuk-visually-hidden">{`Enter details for tap ${index + 1}`}</legend>
+        <div className="flex-container">
+            <div className="govuk-!-margin-left-4 govuk-!-margin-right-2">
+                <FormGroupWrapper errors={errors} errorIds={[`multi-tap-number-${index}`]} hideErrorBar>
+                    <>
+                        <label className="govuk-label" htmlFor={`multi-tap-number-${index}`}>
+                            <span className="govuk-label" id={`multi-tap-number-hint-${index}`}>
+                                Tap Number &nbsp;
+                            </span>{' '}
+                        </label>
+                        <FormElementWrapper
+                            errors={errors}
+                            errorId={`multi-tap-number-${index}`}
+                            errorClass="govuk-input--error"
+                            hideText
+                            addFormGroupError={false}
+                        >
+                            <label className="govuk-label" htmlFor={`multi-tap-number-${index}`}>
+                                <span className="govuk-visually">{index + 1}</span>
```

Comment:
Could we delete this label and move {index + 1} to the 'Tab number' span? I think it looks better if its all on one line.